src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Schemas/JSON/template.json

```diff
@@ -606,9 +603,30 @@
                 "description": "NuGet package restore - restores NuGet packages for all primary outputs with file extensions *proj or sln",
                 "properties": {
                   "actionId": {
-                    "enum": [ "210D431B-A78B-4D2F-B762-4ED3E3EA9025" ]
+                    "enum": [
+                      "210D431B-A78B-4D2F-B762-4ED3E3EA9025"
+                    ]
                   },
-                  "args": {}
+                  "args": {
```

Comment:
does it only apply for restore post action?
some other post actions are also using similar: 
- https://github.com/dotnet/templating/wiki/Post-Action-Registry#change-file-permissions
- https://github.com/dotnet/templating/wiki/Post-Action-Registry#add-projects-to-a-solution-file

Reply:
Those two have their own definitions above/below this one.